Makefile

```diff
@@ -0,0 +1,60 @@
+# Image URL to use all building/pushing image targets
+IMG ?= quay.io/nmagnezi/observatorium-operator:latest
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+# Install CRDs into a cluster
+install: manifests
+	kustomize build config/crd | kubectl apply -f -
```

Comment:
is there something specific why we need to use kustomize for this?

Reply:
No need, we got it from kube-builder. Removed.